zap/src/main/java/org/zaproxy/zap/view/ZapSupportDialog.java

```diff
@@ -177,4 +190,57 @@ private JButton getBtnCopy() {
         }
         return btnCopy;
     }
+
+    private JButton getBtnSaveSbom() {
+        if (btnSaveSbom == null) {
+            btnSaveSbom = new JButton();
+            btnSaveSbom.setText(Constant.messages.getString("support.savesbom.button"));
+            btnSaveSbom.setToolTipText(
+                    Constant.messages.getString("support.savesbom.button.tooltip"));
+            btnSaveSbom.addActionListener(
+                    e -> {
+                        WritableFileChooser fc = new WritableFileChooser();
```

Comment:
I dunno if or how much it matters but I guess this could still be JFileChooser on the left? (If I recall correctly)

Reply:
It could be .. but it doesnt really make any difference, does it?

Reply:
🤷‍♂️ honestly I'm hoping @thc202 will chime in 😀

Reply:
It's fine either way.